packages/disputer/index.js

```diff
@@ -99,13 +98,22 @@ async function run({
       pollingDelay,
       errorRetries,
       errorRetriesTimeout,
-      priceFeedConfig: customPricefeedConfig,
+      priceFeedConfig,
       disputerConfig,
       disputerOverridePrice
     });
 
     // Client and dispute bot.
-    const empClient = new ExpiringMultiPartyClient(logger, getAbi("ExpiringMultiParty"), web3, empAddress);
+    const empClient = new ExpiringMultiPartyClient(
+      logger,
+      getAbi("ExpiringMultiParty"),
+      web3,
+      empAddress,
+      collateralTokenDecimals,
```

Comment:
Shouldn't the `collateralTokenDecimals == syntheticTokenDecimals` for all future  EMP's, if so then why pass both of them in here

Reply:
The whole point of these changes is to make it both forwards and backward compatible. to accommodate this the CLient needs to know: a) collateralTokenDecimals. b) syntheticTokenDecimals and c) priceFeedDecimals.